srcalc/src/test/java/gov/va/med/srcalc/web/view/InputParserVisitorTest.java

```diff
@@ -19,25 +19,6 @@
  */
 public class InputParserVisitorTest
 {
-    @Test
```

Comment:
Instead of dropping the tests, how about ensuring that a Value is simply not created?